static/report/js/loopalyzer.js

```diff
@@ -0,0 +1,52 @@
+// Moves backward one day
+function loopalyzerBackward() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['loopalyzerBackward' is defined but never used. (no-unused-vars)](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)